java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -270,6 +270,23 @@ CompletableFuture<Boolean> pexpireAt(
      */
     CompletableFuture<Long> ttl(String key);
 
+    /**
+     * Removes the existing timeout on <code>key</code>, turning the <code>key</code> from volatile (a
```

Comment:
We do that because it refers to the argument in the command.  The backticks don't work for java :( 
Unfortunately, each language has their own particulars for javadocs.

Reply:
No I understand, what I meant is that even in Python and Java, we don't wrap every mention of key with ``

Reply:
why not?

Reply:
Another way to look at this:  if the key is formatted differently, then the reader may (wrongly) presume that the we are referring to a different key.